UPGRADE-6.1.md

```diff
@@ -29,3 +29,5 @@ Validator
 ---------
 
  * Deprecate `Constraint::$errorNames`, use `Constraint::ERROR_NAMES` instead
+ * Implementing the `ConstraintViolationInterface` or `ConstraintViolationListInterface`
+   without implementing the `__toString()` method is deprecated. The method will be added to the interface in 7.0.
```

Comment:
The 2nd sentence should be removed (and the final dot too)

Reply:
Done